wurst/_handles/Unit.wurst

```diff
@@ -407,3 +407,93 @@ public function unit.getDefaultAquireRange() returns real
 
 public function unit.getDefaultFlyHeight() returns real
 	return GetUnitDefaultFlyHeight(this)
+
+public function unit.getMaxHP() returns int
```

Comment:
name already exists. Do you know the difference between this and `UNIT_STATE_MAX_LIFE` ?

Reply:
Returns an int rather than int.